.env.prod

```diff
@@ -0,0 +1,27 @@
+APP_NAME=StrideForEducation
+
+APP_ENV=PRODUCTION
+API_SECRET_KEY="supersecretkey"
+API_ALLOWED_HOSTS=".localhost 127.0.0.1 [::1]"
+
+POSTGRES_HOST=db
+POSTGRES_NAME=postgres
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=password
+POSTGRES_PORT=5432
+
+DJANGO_SUPERUSER_PASSWORD=Password123
+DJANGO_SUPERUSER_EMAIL=csf@csf.com
+DJANGO_SUPERUSER_USERNAME=admin
+
+EMAIL_HOST='smtp.sendgrid.net'
+EMAIL_PORT=587
+EMAIL_HOST_USER='apikey'
+EMAIL_USE_TLS=True
+EMAIL_ADDRESS_FROM='test@codersforcauses.com'
+SENDGRID_API_KEY=''
+
+JWT_SIGNING_KEY="communityspiritfoundation"
+
+FRONTEND_URL="http://localhost:8082"
+VITE_BACKEND_URL="http://localhost:8081/api"
```

Comment:
We dont want to push the production env file - I'll Add the variables on the env.prod file in the digital ocean server

Reply:
> We dont want to push the production env file - I'll Add the variables on the env.prod file in the digital ocean server

ok, well then I reverted to the initial env dev, so I guess we will add the correct variables in later stages